Packs/ManageEngine/README.md

```diff
@@ -0,0 +1,2 @@
+ManageEngine Endpoint Central is a Unified Endpoint Management solution that helps in managing thousands of servers, desktops, laptops and mobile devices from a single console.
+This integration collects events from the ManageEngine API.
```

Comment:
```suggestion
```
This is the general readme of the pack, so no need to specify the specific integration